### ISSUE TITLE
Failing test(s): TestAccLookerInstance_*

### ISSUE THREAD

**Issue by slevenick** [hashicorp/terraform-provider-google#17592]
### Impacted tests

- TestAccLookerInstance_lookerInstanceCustomDomainExample

### Affected Resource(s)

- google_looker_instance

### Failure rates

- 100% since 2024-02-23
- 100% (20 failed out of last 20 runs) as of 2024-03-14

### Message(s)

```
Error waiting for Creating Instance: Error code 13, message: an internal error has occurred
```

### Nightly build test history

https://hashicorp.teamcity.com/test/-7727801287436741311?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS&expandTestHistoryChartSection=true

b/330175267

**Comment by efeelaiho** [comment]
This is our documentation for custom domain: http://cloud/looker/docs/looker-core-custom-domain. It is expected behavior that the custom domain can only be added after instance creation. We do not plan to implement instance creation + custom domain as a single apply step. 

So essentially we cannot expect this test to function in its current behavior. More than likely needs to be removed.

**Comment by nkhasuntsev** [comment]
Hi @efeelaiho ! Thanks for taking the time to have a look at this one. What's interesting is that the test worked fine when it was initially created in the og PR (https://github.com/GoogleCloudPlatform/magic-modules/pull/9936), and the custom domain is supported by the Rest API (https://cloud.google.com/looker/docs/reference/rest/v1/projects.locations.instances#CustomDomain). We also already tested the custom_domain configuration in our Looker deployments and though we ran into a weird situation during domain verification (still investigating this one), the custom domain was created during instance creation.
